wgpu-core/src/track/mod.rs

```diff
@@ -1,7 +1,7 @@
 /*! Resource State and Lifetime Trackers
 
 These structures are responsible for keeping track of resource state,
-generating barriers where needed, and making sure resources are kept
+generating barriers where needednd making sure resources are kept
```

Comment:
nit: looks like the next few comment modifications are accidental.

Reply:
oops, one of my many regex replacements must have gone sideways!